lib/objdata/suitsets

```diff
@@ -6,31 +6,45 @@
 :head     = <vnum>
 :neck     = <vnum>
 :body     = <vnum>
+:back     = <vnum>
 :arm      = <vnum>
+:wrist    = <vnum>
 :hand     = <vnum>
+:finger   = <vnum>
 :waist    = <vnum>
-:wrist    = <vnum>
 :leg      = <vnum>
 :foot     = <vnum>
-:finger   = <vnum>
 :shield   = <vnum>
-:back     = <vnum>
 :Set #
 :#
 :$
 :head     = 
 :neck     = 
 :body     = 
+:back     = 
 :arm      = 
+:wrist    = 
 :hand     = 
+:finger   = 
 :waist    = 
-:wrist    = 
 :leg      = 
 :foot     = 
-:finger   = 
 :shield   = 
-:back     = 
 // Paste below this line.
+// ----------------------
+:Set #158
+:#Flowing Black
+:$Human
```

Comment:
Others use lowercase here. I don't know if it matters, but uniformity would be good too